packages/docusaurus-theme-classic/src/theme-classic.d.ts

```diff
@@ -611,8 +612,7 @@ declare module '@theme/MDXComponents' {
     readonly h4: (props: ComponentProps<'h4'>) => JSX.Element;
     readonly h5: (props: ComponentProps<'h5'>) => JSX.Element;
     readonly h6: (props: ComponentProps<'h6'>) => JSX.Element;
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    [tagName: string]: ComponentType<any>;
```

Comment:
We need this because users may wrap `MDXComponents` and introduce more components. IIRC it's also needed by `mdx-js/react`, because last time I removed this index signature there was an error

Reply:
I understand, but I don't know how to fix improper types for new `admonition`, so I temporarily removed this.